bpf/unwinders/native.bpf.c

```diff
@@ -1110,6 +1115,22 @@ static __always_inline bool set_initial_state(struct bpf_perf_event_data *ctx) {
   unwind_state->stack_key.user_stack_id = 0;
   unwind_state->stack_key.kernel_stack_id = 0;
   unwind_state->stack_key.interpreter_stack_id = 0;
+  unwind_state->stack_key.trace_id[0] = 0;
+  unwind_state->stack_key.trace_id[1] = 0;
+  unwind_state->stack_key.trace_id[2] = 0;
+  unwind_state->stack_key.trace_id[3] = 0;
+  unwind_state->stack_key.trace_id[4] = 0;
+  unwind_state->stack_key.trace_id[5] = 0;
+  unwind_state->stack_key.trace_id[6] = 0;
+  unwind_state->stack_key.trace_id[7] = 0;
+  unwind_state->stack_key.trace_id[8] = 0;
+  unwind_state->stack_key.trace_id[9] = 0;
+  unwind_state->stack_key.trace_id[10] = 0;
+  unwind_state->stack_key.trace_id[11] = 0;
+  unwind_state->stack_key.trace_id[12] = 0;
+  unwind_state->stack_key.trace_id[13] = 0;
+  unwind_state->stack_key.trace_id[14] = 0;
+  unwind_state->stack_key.trace_id[15] = 0;
```

Comment:
I would think a bzero or memset call would be more efficient here but I haven't investigated it.